app/lib/common/utils/utils.dart

```diff
@@ -70,8 +71,27 @@ extension RefDebounceExtension on Ref {
 
 const largeScreenBreakPoint = 770;
 
-bool isLargeScreen(BuildContext context) {
-  return MediaQuery.of(context).size.width >= largeScreenBreakPoint;
+extension ActerContextUtils on BuildContext {
+  bool get isLargeScreen =>
+      MediaQuery.of(this).size.width >= largeScreenBreakPoint;
+
+  NavigatorState closeDialog({Routes? orGo, bool ignorePopFailure = false}) {
```

Comment:
Not understanding the reason for `ignorePopFailure`. Why we need to ignore that?

Reply:
Yes I have read it bit confused over there.